src/mac_platform.m

```diff
@@ -34,25 +34,23 @@ bool platform_create_window_objc(int width, int height, char* title)
 
 void platform_update_window_objc()
 {
-    while (running) {
```

Comment:
Fixed bug where window held program execution unnecessarily. Now execution gets back to main loop.